1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,29 @@
-function readingList(books) {
-  // Write your code here...
+
+
+
+function readingList(books) { 
```

Comment:
Well written code! The variables are all well-named and easy to understand. Great job!